MovieStoreWebApi/Applications/GenreOperations/Commands/DeleteGenre/DeleteGenreCommand.cs

```diff
@@ -4,18 +4,20 @@ namespace MovieStoreWebApi.Applications.GenreOperations.Commands.DeleteGenre
 {
     public class DeleteGenreCommand
     {
-        public int Id { get; set; }
+        public int GenreId { get; set; }
         private readonly IMovieStoreDbContext _context;
+        public const string ExceptionMessage = "Tür bulunamadi.";
```

Comment:
Farklı tip accessore vs. sahip fieldların arasında bir satır boşluk bırakmak okurken karışıklığı azaltır. yani public ve private field arasında boşluk bırakmak, const field arasında boşluk bırakmak iyi olur. Benzer türler de bir arada bulunduğunda bu şekilde yazmak daha da iyi oluyor. Yani burada 7-8-9 satırlarındaki fieldlar farklı türlerde olduğu için aralarında boşluk bırak.

Reply:
Düzeltildi.